rollout/trafficrouting/istio/istio_types.go

```diff
@@ -30,23 +48,23 @@ type VirtualServiceHTTPRouteDestination struct {
 
 // Destination fields within the VirtualServiceDestination struct of the Virtual Service that the controller modifies
 type VirtualServiceDestination struct {
-	Host   string `json:"host,omitempty"`
+	Host string   `json:"host,omitempty"`
```

Comment:
go fmt?

Reply:
Sorry I forgot to run `go fmt` before pushing the changes. Will run and update the commit. Thanks!